### ISSUE TITLE
Important info.

### ISSUE THREAD

**Issue by Master-Foo** [Cabal-Syndicate/randora_server#27]
Some things to be aware of. Comment on this message when you read it so I can close it.

Pull to latest version.

The fix for the Firefox issue is that Firefox is picky about end tags in HTML. For instance

``` html
<div/>
```

is generally valid, however certain elements (like div) need to have a verbose end tag like this:

``` html
<div></div>
```

The XML library doesn't differentiate between tags which can be short and tags which need to be verbose
So, the fix is to add "nothing" to the tag to fool the XML library into thinking there is something in it. For instance:

``` d
class Asdf : DivElement{
    this.tag.attr["class'] = "foo";
}
```

will print out this:

``` html
<div class="foo"/>
```

Which is bad...

However...

``` d
class Asdf : DivElement{
    this.tag.attr["class'] = "foo";
    this ~= new Text(""); //insert "nothing" into the tag.
}
```

will print out this:

``` html
<div class="foo"></div>
```

Which is what Firefox wants. Otherwise, it will throw a fit and render the page wrong.

So, when writing code, just remember that there needs to be "something" in an element. Even if that "something" is "nothing".

---

Next Item...

Some code has been changed. The particulars are irrelivant to you at this time. However, one thing you need to know is that when you:

``` d
void init(){
    //Do Stuff
}
```

now do:

``` d
override void init(){
    //Do Stuff
}
```

In other words just add "override in front of `void init()`. Don't worry about "WHY". It's above your pay grade right now. I'll tell you when you are older.
